pastepwn/core/tests/paste_test.py

```diff
@@ -49,10 +49,17 @@ def test_set_body(self):
         my_body = "This is a test for pastepwn"
         self.paste.set_body(my_body)
         self.assertEqual(my_body, self.paste.body)
+        empty_body = ""
+        self.paste.set_body(None)
+        self.assertEqual(empty_body, self.paste.body) 
 
     def test_to_dict(self):
         my_body = "This is a test for pastepwn"
         self.paste.set_body(my_body)
 
         paste_dict = self.paste.to_dict()
         self.assertEqual(self.p, paste_dict)
+        
+    def test_str(self):
+        string_of_dict = str(self.p)
+        self.assertEqual(string_of_dict, str(self.paste))
```

Comment:
Although in general I like the Idea, we should remove this, since a dict is an unordered datastructure. Hence calling the str() function on it can lead to different representations/orders of the dict.